config.go

```diff
@@ -25,6 +29,16 @@ func LoadConfigFromPath(path string) (Config, error) {
 	if dsn, found := os.LookupEnv("TIGERBLOOD_DSN"); found {
 		config.DatabaseDsn = dsn
 	}
+	if dbMaxOpenConns, found := os.LookupEnv("TIGERBLOOD_DB_MAX_OPEN_CONNS"); found {
+		dbMaxOpenConns, err := strconv.Atoi(dbMaxOpenConns)
+		if err != nil {
+			return config, err
+		}
+		config.DatabaseMaxOpenConns = dbMaxOpenConns
+	}
+	if config.DatabaseMaxOpenConns == 0 {
+		config.DatabaseMaxOpenConns = defaultDatabaseMaxOpenConns
```

Comment:
This line can be moved above`if dbMaxOpenConns, found := os.LookupEnv(`... and then you can remove the if statement.